scarb/src/core/package/name.rs

```diff
@@ -88,9 +88,89 @@ impl PackageName {
             }
         }
 
+        if PackageName::is_keyword(&name) {
+            bail!(
+                "the name `{name}` cannot be used as a package name, \
+                names can't use cairo keywords see the full list at \
```

Comment:
```suggestion
                names cannot use Cairo keywords see the full list at \
```

1. avoid short forms
2. Cairo is a proper name, please capitalize it